src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -137,7 +137,7 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
                                             isInline
                                         >
                                             {complianceFetchStatus === 'fulfilled' && Array.isArray(complianceSummary.data) &&
-                                                3 - complianceSummary.data.length >= 1 &&
+                                                complianceSummary.data.length - 3 >= 1 &&
                                                     `${3 - complianceSummary.data.length} more compliance reports`
```

Comment:
Doesn't this also need to be updated?

```js
`${complianceSummary.data.length - 3} more compliance reports`
```

Reply:
No, this one is accurate - if you have 5, 5-3 -> 2 more compliance reports. If you have 3 or less, it won't show up at all.

Reply:
If you have 5, it's `3-5 -> -2`. What am I missing here?

Reply:
I was reading the condition wrong - sorry about that, it's the same